doc/sphinx/source/recipes/recipe_iht_toa.rst

```diff
@@ -0,0 +1,123 @@
+.. _recipes_iht_toa:
+
+Implied heat transport from Top of Atmosphere fluxes
+====================================================
+
+Overview
+--------
+
+This recipe calculates the implied horizontal heat transport (IHT) due to the
+spatial anomalies of radiative fluxes at the top of the atmosphere (TOA).
+The regional patterns of implied heat transport for different components of
+the TOA fluxes are calculated by solving the Poisson equation with the flux
+components as source terms.
+It reproduces the plots in `Pearce and Bodas-Salcedo (2023)`_ when the input
+data is CERES EBAF.
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in esmvaltool/recipes/
+
+    * recipe_iht_toa.yml calculates the IHT maps for the following radiative fluxes:
+
+        * Total net, SW net, LW net (Figure 2).
+        * Total CRE, SW CRE, LW CRE (Figure 4).
+        * All-sky and clear-sky reflected SW (Figure 5).
+        * The meridional heat transports (MHT) of the fluxes above (Figures 1 and 3).
```

Comment:
I recognise that this is how it is formatted in the template; I have opened #3377 to address this :)

Reply:
Changed as suggested, it looks OK now.

Reply:
The second set of points are still showing as a quote at https://esmvaltool--3177.org.readthedocs.build/en/3177/recipes/recipe_iht_toa.html#available-recipes-and-diagnostics. Since I'm adding a suggestion that will fix this I will resolve this conversation :)